### ISSUE TITLE
Train SFT models

### ISSUE THREAD

**Issue by 0x000011b** [PygmalionAI/logbooks#2]
Lots of the papers we take inspiration from didn't use a regular CLM loss objective for fine-tuning their models, but instead did supervised fine-tuning using only the response tokens to calculate loss. I'd like to try this out and see how it compares to our  UFT runs.

**Comment by 0x000011b** [comment]
Keeping track [here](https://github.com/PygmalionAI/logbooks/blob/master/2023-01-14.md)!

**Comment by 0x000011b** [comment]
Released a couple SFT models but they don't seem to be a clear improvement compared to the previous runs. Will consider this as done for now and focus on trying to improve the models by taking a closer look at the training data instead.
